hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/OMAdminProtocol.java

```diff
@@ -39,4 +43,23 @@ public interface OMAdminProtocol extends Closeable {
    * Remove OM from HA ring.
    */
   void decommission(OMNodeDetails removeOMNode) throws IOException;
+
+  /**
+   * Asynchronously reload configuration on disk and apply changes.
+   */
+  @Idempotent
+  void startOmReconfiguration() throws IOException;
+
+  /**
+   * Get the status of the previously issued reconfig task.
+   * @see org.apache.hadoop.conf.ReconfigurationTaskStatus
+   */
+  @Idempotent
+  ReconfigurationTaskStatus getOmReconfigurationStatus() throws IOException;
+
+  /**
+   * Get a list of allowed properties for reconfiguration.
+   */
+  @Idempotent
+  List<String> listOmReconfigurableProperties() throws IOException;
```

Comment:
Please define these in a separate interface (without reference to `Om`) and let `OMAdminProtocol` extend that interface.